1.0-Upgrade.md

```diff
@@ -0,0 +1,106 @@
+
+The following tables provide a mapping of the previous flags we supported and their new version (if applicable).
+
+#### krane deploy
+
+Old flag | New flag | Comments
+--- | --- | ---
+--bindings=BINDINGS | --bindings=BINDINGS |
+--skip-wait | --verify-result=true |
+--allow-protected-ns | --protected-namespaces=default,kube-system,kube-public | Added the ability to specify which namespaces are protected
+--no-prune | --prune=true |
+--template-dir | -f, --filename | Makes all our commands accept this arg, which is now required for the deploy task
+--verbose-log-prefix | --verbose-log-prefix |
+--max-watch-seconds=seconds | --global-timeout=300s | Changed flag name and default value
+--selector | --selector |
+-h, --help | -h, --help |
+-v, --version | [none] | Replaced with `krane version`
+$ENVIRONMENT | [none] | Dropped in favour of `-f`
+$REVISION | --current-sha | The environment variable REVISION was dropped in favour of an explicit flag
+[none] | --render-erb | **Important:** the new CLI doesn't render ERB by default
+
+#### krane restart
+
+Old flag | New flag | Comments
+--- | --- | ---
+--deployments=LIST | --deployments=LIST |
+--max-watch-seconds=seconds | --global-timeout=300s | Changed flag name and default value
+[none] | --verify-result=true | Define whether it should wait for results or exit immediately after validation
+
+#### krane run
+
+Old flag | New flag | Comments
+--- | --- | ---
+--skip-wait | --verify-result=true |
+--max-watch-seconds=seconds | --global-timeout=300s | Changed flag name and default value
+--entrypoint | --command | Changed flag name to make its purpose clearer
+--template | --template | Changed to be required
+[it is positional now] | --arguments | Optional flag, as `command` or the template might already specify the required arguments
+--env-vars=ENV_VARS | --env-vars=ENV_VARS |
+
+#### krane render
+
+Old flag | New flag | Comments
+--- | --- | ---
+--bindings=BINDINGS | --bindings=BINDINGS |
+--template-dir | -f, --filename | Changed to be more aligned with `kubectl apply` and our other tasks
+$REVISION | --current-sha | The environment variable REVISION was dropped in favour of an explicit flag
+
+## Running `kubernetes-deploy` and `krane` side by side
+
+If you attempt to install two gems that have conflicting executables (as is the case here), `gem install` will warn you but the most recently installed one will win. This means that you can run both `kubernetes-deploy` 0.30.0 and `krane` 1.0.0 side by side by doing:
+
+```bash
+gem install kubernetes-deploy -v 0.30.0
+gem install krane -v 1.0.0
+```
+
+This can help you incrementally port scripts that use the old CLI to the new one.
+
+## New task: `krane global-deploy`
+
+`krane global-deploy` (accessible through the Ruby API as `Krane::GlobalDeployTask`) can deploy global (non-namespaced) resources such as `PersistentVolume`, `Namespace`, and `CustomResourceDefinition`. Its interface is very similar to `krane deploy`. Example usage:
```

Comment:
Should mention that `krane deploy` does not support global resources? Maybe add the same comment in the `krane deploy` as well?